rpc/cloudagent/cloud/etcdcluster/moc_cloudagent_etcdcluster.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/cloud";
+package moc.cloudagent.etcd;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+import "moc_cloudagent_etcdserver.proto";
+
+message EtcdClusterRequest {
+	repeated EtcdCluster EtcdClusters = 1;
+	Operation OperationType = 2;
+}
+
+message EtcdClusterResponse {
+	repeated EtcdCluster EtcdClusters = 1;
+	google.protobuf.BoolValue Result = 2;
+	string Error = 3;
+}
+
+message EtcdCluster {
+	string id = 1;
+	string name = 2;
+	string locationName = 3;
+	string groupName = 4;
+	// etcd ca certificate that works as RoT for client and peer connections
+	string caCertificate = 5;
+	// etcd ca key associated with the ca certificate
+	string caKey = 6;
+	repeated EtcdServer servers = 7;
```

Comment:
This might not be needed.
If you keep it here, anytime a new server is added, we have to update this in 2 places in cloud agent.

Do you need it or you added it because secrets was under keyvault(not sure why secrets is listed under keyvault - might have to be removed as well)

Reply:
Probably do not need it, was mirroring keyvault/secrets. Ok, will simplify.

Reply:
awesome. Thanks